js/viz/chart.js

```diff
@@ -7,7 +7,8 @@ import {
     map as _map, getLog, getCategoriesInfo,
     updatePanesCanvases, convertVisualRangeObject, PANE_PADDING,
     normalizePanesHeight,
-    checkElementHasPropertyFromStyleSheet
+    checkElementHasPropertyFromStyleSheet,
+    rangesIsEqual
```

Comment:
Is it ok that plural 'ranges' goes with singular 'is', should it be rangesAreEqual? 

Or is it by design?